models/intel/speech-to-text-en-0001/accuracy-check.yml

```diff
@@ -0,0 +1,24 @@
+models:
+  - name: speech-to-text-en-0001
+    launchers:
+      - framework: dlsdk
+        allow_reshape_input: True
+        adapter:
+          type: ctc_greedy_decoder
+          alphabet: ["<PAD>", "<UNK>", "<BOS>", "<EOS>", "_", "e", "t", "a", "o", "n", "i", "h", "s", "r", "d", "l", "u", "m", "c", "w", "f", "g", "y", "p", "b", "v", "k", "x", "j", "q", "z", "_t", "he", "_a", "_the", "in", "_s", "_w", "_o", "re", "nd", "_b", "_h", "er", "_m", "_i", "ou", "_c", "_f", "at", "ed", "_and", "en", "_to", "_of", "on", "is", "_d", "ing", "_th", "_p", "_he", "or", "es", "_l", "_in", "as", "ll", "it", "ar", "an", "_n", "_g", "om", "_be", "_ha", "_e", "le", "ot", "_y", "ut", "ow", "ic", "_wh", "_it", "ld", "ve", "_that", "ly", "_was", "id", "se", "st", "_on", "gh", "ent", "_re", "_you", "im", "ce", "_u", "ver", "ion", "_as", "et", "_for", "ay", "_his", "ith", "_we", "al", "ir", "_r", "_with", "_st", "ad", "ur", "ght", "_an", "_her", "_not", "_had", "_is", "ter", "her", "ac", "am", "_at", "oo", "_but", "ould", "_she", "_k", "_se", "_sa", "_sh", "_fr", "_him", "_so", "ill", "_me", "ain", "_su", "ight", "ch", "red", "ct", "all", "ro", "ess", "ke", "il", "ore", "_de", "_they", "_my", "_whe", "_all", "ich", "_ne", "ri", "_by", "_have", "ome", "pp", "_this", "_li", "_do", "_con", "us", "_which", "_ch", "ul", "qu", "_j", "nt", "_up", "_said", "_from", "ard", "ge", "_or", "ant", "_v", "_were", "_no", "th", "_ex", "_one", "res", "pe", "and", "est", "_who", "ble", "_man", "_there", "_al", "ie", "ous", "ust", "very", "ers", "ation", "_fe", "_them", "lf", "_when", "ind", "ame", "ra", "_go", "ast", "ood", "_kn", "fe", "_int", "ist", "art", "_are", "out", "_would", "os", "_le", "_their", "ong", "_what", "our", "_if", "ound", "_com", "_ab", "_out", "_wor", "em", "_will", "ak", "_mis", "ol", "ate", "um", "un", "itt", "ough", "ked", "ap", "ig", "one", "_been", "own", "ive", "if", "_then", "_br", "_ar", "_tr"]
```

Comment:
wow, maybe better to use file as vocab instead of providing in config?

Reply:
@eaidova I agree, but I didn't want to break backward compatibility. And I think that it more lightweight for accuracy_checker to use default configs to store alphabet.
But I use .json format for the demo app like:
{
"vocab": [...],
"space_symbol": '_'
"pad_id": pad_id,
"bos_id": bos_id,
"eos_id": eos_id,
"unk_id": unk_id
}
I can use this file in accuracy_checker too. Is it good for you?
How we should pass the vocab_path to accuracy checker? Should we use existed alphabet variable (if variable is not list, we try to open vocab file or something like that)? Thanks!

Reply:
probably you can, but it will require to do additional step in configuration (resolve prefix for path), so maybe to have `vocab` for inplace vocab in config and `vocab_file` for vocab represented as file and check that one or another defined in config